v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/writer/DeadLetterQueue.java

```diff
@@ -169,7 +169,7 @@ protected FailsafeElement<String, String> rowContextToDlqElement(RowContext r) {
       json.put(f.name(), value == null ? null : value.toString());
     }
     return FailsafeElement.of(json.toString(), json.toString())
-        .setErrorMessage("TransformationFailed: " + String.valueOf(r.err()));
+        .setErrorMessage("TransformationFailed: " + r.err() + "\n" + r.getStackTraceString());
```

Comment:
it would be better to have a different field called error stacktrace and just save this there

Reply:
Hmm but we already have that in our err throwable, storing twice would be redundant right